src/pages/LessonPage.tsx

```diff
@@ -33,14 +34,20 @@ export default function LessonPage(props: LessonPageProps) {
   return (
     <Container>
       {props.blocks[currentBlock]}
-
-      <LessonButton
-        onClick={() =>
-          setCurrentBlock((cur) => (cur + 1) % props.blocks.length)
-        }
-      >
-        Next
-      </LessonButton>
+      <Box sx={{ display: 'flex', justifyContent: 'space-between' }}>
+        <LessonButton
+          onClick={() => setCurrentBlock((cur) => (cur - 1 >= 0 ? cur - 1 : 0))}
```

Comment:
this is a nit, but can you make `cur` a more descriptive name like `blockIndex` (if that's what it is), because I'm only guessing right now

Reply:
Well I was actually copying what Asir did, but yeah I can change to blockIndex